prody/ensemble/functions.py

```diff
@@ -401,7 +424,9 @@ def buildPDBEnsemble(refpdb, PDBs, title='Unknown', labels=None, seqid=94, cover
         for chain in refchains:
             mappings = mapping_func(pdb, chain,
                                     seqid=seqid,
-                                    coverage=coverage)
+                                    coverage=coverage,
+                                    alignment=alignment_list[i],
+                                    alignments=alignments_list[i])
```

Comment:
I think hiding alignment and alignments in kwargs is a good idea because these arguments are not required for every mapping functions.